packages/clerk-js/src/ui/utils/colors/colorOptionToHslaScale.ts

```diff
@@ -0,0 +1,245 @@
+import type { ColorScale, CssColorOrAlphaScale, CssColorOrScale, HslaColor, HslaColorString } from '@clerk/types';
```

Comment:
is this file different from `packages/clerk-js/src/ui/utils/colorOptionToHslaScale.ts`?